sensor/intrinsic_camera_calibrator/intrinsic_camera_calibrator/intrinsic_camera_calibrator/board_detectors/apriltag_grid_detector.py

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env python3
+
+# Copyright 2022 Tier IV, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from collections import defaultdict
+
+from dt_apriltags import Detector as _Detector
```

Comment:
Sorry, the commit I was testing was a bit out of date. I'm getting an import error here now.
```
[camera_calibrator-1]     from dt_apriltags import Detector as _Detector
[camera_calibrator-1] ModuleNotFoundError: No module named 'dt_apriltags'
```

Reply:
Ah, sorry. Dependent packages were missing in my environment.